src/server/game/Entities/Creature/CreatureData.h

```diff
@@ -389,6 +406,7 @@ struct CreatureModelInfo
     float combat_reach;
     uint8 gender;
     uint32 modelid_other_gender;
+    float is_trigger;
```

Comment:
the value for is_trigger is taken from dbc and can't be set in db right?

Reply:
I don't know of anything in DBC that would determine a trigger model, nor does the implementation aim to mark models as triggers in DB, it uses a hardcoded list [here](https://github.com/azerothcore/azerothcore-wotlk/pull/19068/files#diff-59e59afabb9e07b951f9cd43dc7ef08e8c894b2aa889ae394fc4ace2eef09242R1786).

Reply:
Right I saw that. I blame 1.30am brain

Reply:
We could check for the `CREATURE_FLAG_EXTRA_TRIGGER`

Reply:
> We could check for the `CREATURE_FLAG_EXTRA_TRIGGER`

We might be able to, but I personally prefer this approach as (with proper data to work with) it doesn't require us to go through each and every creature and add a single flag if they use a certain model. IMO core list is simpler.